packages/quantic/force-app/main/default/lwc/quanticModal/quanticModal.js

```diff
@@ -94,4 +103,51 @@ export default class QuanticModal extends LightningElement {
   get tabindex() {
     return this.isVisible ? 0 : -1;
   }
+
+  get allAssignedElements() {
+    const headerSlot = this.template.querySelector('slot[name=header]');
+    const contentSlot = this.template.querySelector('slot[name=content]');
+    const footerSlot = this.template.querySelector('slot[name=footer]');
+
+    const allAssignedElements = [
+      // @ts-ignore
+      ...headerSlot.assignedElements(),
+      // @ts-ignore
+      ...contentSlot.assignedElements(),
+      // @ts-ignore
+      ...footerSlot.assignedElements(),
+    ].map((element) => Array.from([element, ...element.childNodes]));
+    return this.flattenArray(allAssignedElements);
+  }
+
+  focusOnLastElement() {
+    for (const element of this.allAssignedElements.reverse()) {
```

Comment:
Is this safe if the allAssignedElements is empty?

Reply:
Can't you do this with a `find` ?

```
const focusableElement = this.allAssignedElements.reverse().find(element => !!getLastFocusableElement(element));
if(focusableElement) {
  focusableElement.focus();
}
```

Not 100% sure this works just out of my head ;)

Reply:
it would work with a small adjustment:
```
    const focusableElement = this.allAssignedElements
       .reverse()
       .find(element => !!getLastFocusableElement(element));
    if (focusableElement) {
      getLastFocusableElement(focusableElement).focus();
    }
```